packages/terra-dropdown-button/src/terra-dev-site/doc/example/DefaultSplitButton.jsx

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-console */
+import React from 'react';
+
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import { SplitButton, Button } from 'terra-dropdown-button/lib/DropdownButton';
+
+export default () => (
+  <SplitButton
+    primaryOptionLabel="Primary option"
+    onClick={() => { console.log('primary option'); }}
+  >
+    <Button label="Abacadabra" onClick={() => { console.log('hi'); }} />
```

Comment:
Let's update this example to write content to the page instead of a console.log.